auth-service/src/main/kotlin/uk/gov/communities/delta/auth/controllers/internal/AdminEditEmailController.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.communities.delta.auth.controllers.internal
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.slf4j.LoggerFactory
+import uk.gov.communities.delta.auth.plugins.ApiError
+import uk.gov.communities.delta.auth.services.DeltaSystemRole
+import uk.gov.communities.delta.auth.services.UserLookupService
+import uk.gov.communities.delta.auth.services.UserService
+
+class AdminEditEmailController(
```

Comment:
Maybe `AdminEditUserEmailController`? Just to make it clearer editing a user, not an email